learning-reactjs/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "learning-reactjs",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@atlaskit/button": "^16.5.5",
+    "@atlaskit/icon": "^21.11.5",
+    "@atlaskit/textfield": "^5.3.5",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "styled-components": "^5.3.6",
+    "uuid": "^9.0.0",
+    "web-vitals": "^2.1.4"
+  },
+  "scripts": {
```

Comment:
đọc một xíu về react-scripts: https://blog.logrocket.com/everything-you-need-know-about-react-scripts/